vnext/ReactUWP/Views/ViewViewManager.cpp

```diff
@@ -87,8 +87,14 @@ class ViewShadowNode : public ShadowNodeBase {
   void TabIndex(int32_t tabIndex) {
     m_tabIndex = tabIndex;
 
-    if (IsControl())
-      GetControl().TabIndex(m_tabIndex);
+    if (IsControl()) {
+      if (tabIndex < 0) {
+        GetControl().IsTabStop(false);
```

Comment:
>GetControl().IsTabStop(false); [](start = 8, length = 30)

In addition to setting IsTabStop = false, consider clearing the TabIndex property.

Reply:
This makes sense. Now clearing the property and tested that everything continues to work.

---
In reply to: [385377113](https://github.com/microsoft/react-native-windows/pull/4180#discussion_r385377113) [](ancestors = 385377113)